Misc/NEWS.d/next/Library/2024-09-25-18-07-51.gh-issue-120378.NlBSz_.rst

```diff
@@ -0,0 +1 @@
+Fix a crash related to an integer overflow in :func:`curses.resizeterm`.
```

Comment:
```suggestion
Fix a crash related to an integer overflow in :func:`curses.resizeterm`
and :func:`curses.resize_term`.
```

I think we should indicate that both functions actually take short inputs instead of integers, namely we need to document it (the C signature is (int, int) so it may be confusing).

Reply:
IMO, that would be documentation bloat. From what GNU said, basically nobody has tried to make a terminal this big in however many years that curses has existed.

Reply:
Actually, I had in mind the case when someone incorrectly (and dynamically) computes the inputs. By documenting that we expect short integers, they would know in advance that their integer should be in [-2^15, 2^15-1] (we're not even using an unsigned short!). But maybe you're right about this...

Reply:
This will probably be a temporary limit anyway, we're just waiting on GNU to fix this upstream.